contrib/buildahimage/README.md

```diff
@@ -13,6 +13,7 @@ using the latest Fedora and then Buildah is installed into them:
   * quay.io/buildah/stable - This image is built using the latest stable version of Buildah in a Fedora based container.  Built with buildahimage/stable/Dockerfile.
   * quay.io/buildah/upstream - This image is built using the latest code found in this GitHub repository.  When someone creates a commit and pushes it, the image is created.  Due to that the image changes frequently and is not guaranteed to be stable.  Built with buildahimage/upstream/Dockerfile.
   * quay.io/buildah/testing - This image is built using the latest version of Buildah that is or was in updates testing for Fedora.  At times this may be the same as the stable image.  This container image will primarily be used by the development teams for verification testing when a new package is created.  Built with buildahimage/testing/Dockerfile.
+  * quay.io/buildah/stable:version - This image is built 'by hand' using a Fedora based container.  A rpm is first pulled from the [Fedora Updates System](https://bodhi.fedoraproject.org/) and the image is built from there.  For more information, see the Containerfile used to build it, buildahimage/stablebyhand/Containerfile.buildahstable
```

Comment:
nit: I think this should be `An rpm ...`
At least that is the way I would say it.  But I am not sure.

Reply:
You're right!  The word "an" is supposed to be used before acronyms if the first letter of the acronym makes a vowel sound.  I did not think "R" made a vowel sound, but according to the experts it does.  Will fix up and see what's what with the tests.